readthedocs/projects/models.py

```diff
@@ -570,8 +572,15 @@ def has_aliases(self):
         return self.aliases.exists()
 
     def has_pdf(self, version_slug=LATEST):
+        version = self.versions.filter(slug=version_slug).first()
         if not self.enable_pdf_build:
```

Comment:
@ericholscher, are you about this check, on L573?  I think, I have tried this, that breaks tests.